bazel/toolchain/aarch64-none-linux-gnu/wrappers/aarch64-none-linux-gnu-ar

```diff
@@ -1 +0,0 @@
-wrapper
```

Comment:
Please, remove all those changes

Reply:
@alicefr  i have made changes on the recommdation of @0xFelix( Run make format and amend the changes to your commit. That should solve the issue.)